src/main/java/org/breedinginsight/brapps/importer/controllers/ImportController.java

```diff
@@ -179,6 +179,10 @@
         } catch (AlreadyExistsException e) {
             log.info(e.getMessage());
             return HttpResponse.status(HttpStatus.CONFLICT, e.getMessage());
+        } catch (ValidatorException e) {
+            log.info(e.getMessage());
```

Comment:
Can this be changed to something like `log.info("Validation errors", e);`. That way the error stack gets logged as well, which is very helpful when debugging via the logs

Reply:
Pushed changes.